README.md

```diff
@@ -28,29 +27,32 @@ Fpdart is inspired by [fp-ts](https://gcanti.github.io/fp-ts/), [cats](https://t
 
 ***
 
-- [📖 Learn `functional programming` and `fpdart`](#-learn-functional-programming-and-fpdart)
-  - [👨‍💻 Blog posts and tutorials](#-blog-posts-and-tutorials)
-- [💻 Installation](#-installation)
-- [✨ Examples](#-examples)
-  - [Option](#option)
-  - [Either](#either)
-  - [IO](#io)
-  - [Task](#task)
-  - [Utility types](#utility-types)
-  - [Reader](#reader)
-  - [State](#state)
-  - [📦 Immutable Collections](#-immutable-collections)
-  - [More](#more)
-- [🎯 Types](#-types)
-- [💡 Motivation](#-motivation)
-  - [Goal](#goal)
-  - [Comparison with `dartz`](#comparison-with-dartz)
-- [🤔 Roadmap](#-roadmap)
-- [📃 Versioning](#-versioning)
-- [😀 Support](#-support)
-- [👀 License](#-license)
-
-
+- [`Fpdart`](#fpdart)
```

Comment:
Could you remove this and indent the list of paragraphs without it (see current `README`?

Reply:
I don't know what "this" means here.  I don't understand the red/green diff and how it applies to README.

Reply:
OK, I think I got it.  Just removing from README.md for now.  Add it back if you want.